bundler/lib/bundler/man/bundle-info.1

```diff
@@ -10,7 +10,7 @@
 \fBbundle info\fR [GEM] [\-\-path]
```

Comment:
Maybe we can change to synopsis to use `GEM_NAME` instead? An also the description to be worded with something like

> Given a gem name present in your bundle, print the basic information about it such a homepage, version, path and summary\.

With thise clarifications in place, do you think we still need the explicit example?

Reply:
@deivid-rodriguez I did think of this. Although, I was thinking that people may still try `GEM-VERSION` even if we say name. Hence, I added the example. Though - you folks would know better than me.
Thinking on it a bit more I think `GEM_NAME` is good enough. We can let the users catchup and learn that `GEM_NAME` means without the version. I will go ahead and make the changes once you reply and if you have no other objections.

Reply:
Okay, I agree you with here. I felt there were more reports. I will make changes as you suggested.

Synposis from`GEM` to `GEM_NAME`
Description to 
```
Given a gem name present in your bundle, print the basic information about it such a homepage, version, path and summary.
```

Reply:
There's a lint failure about man pages being out of sync. Can you run `rake man:build` and commit any differences? That should get the build back to green.

Reply:
@deivid-rodriguez Have pushed the same. There were a bunch of changes as all the man pages got updated with current month. Is that what is expected?

Reply:
Yes, that's exactly what was expected. The first documentation update each month also bumps the month in the man pages 👍.